src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ElementType } from 'react'
+import BootstrapBreadcrumbItem from 'react-bootstrap/BreadcrumbItem'
+
+interface Props {
+  /** Defines the attributes of the children items */
+  itemProps?: {
+    as?: ElementType
+    href?: string
+    active?: boolean
+    target?: string
+    title?: string
+  }
+  /** The children to render */
+  children?: React.ReactNode
+}
```

Comment:
I think that we might be able to simplify this interface by doing: 

```ts
interface Props {
  active: boolean
  label: string
  onClick: (event: React.MouseEvent<HTMLAnchorElement>) => void
}

```

Then the implementation may look something like: 

```ts
const BreadcrumbItem = (props: Props) => {
  const { active, label, onClick } = props
  return (
    <BootstrapBreadcrumbItem active={active}>
      <a onClick={onClick}>{label}</a>
    </BootstrapBreadcrumbItem>
  )
}

export { BreadcrumbItem }

```

Reply:
Could we have the implementation like this since `active` prop is will  render children wrapped in `span` rather than `a` tag.  This seems more flexible to me. Thoughts?
```
const BreadcrumbItem = ({ children, active, onClick }: Props) => (
  <BootstrapBreadcrumbItem
    active={active}
    onClick={onClick}>
    {children}
</BootstrapBreadcrumbItem>
)
```

Reply:
Sounds good to me!